packages/stores/types.ts

```diff
@@ -180,6 +213,61 @@ type TClientStore = {
     logout: () => Promise<LogOutResponse>;
     should_allow_authentication: boolean;
     is_crypto: boolean;
+    isEligibleForMoreDemoMt5Svg: (market_type: 'synthetic' | 'financial') => boolean;
+    isEligibleForMoreRealMt5: (market_type: 'synthetic' | 'financial') => boolean;
+    fetchResidenceList?: () => void;
+    account_settings: GetSettings & {
+        upload_file?: string;
+        poi_state?: string;
+    };
+    residence_list: ResidenceList;
+    is_high_risk: boolean;
+    should_restrict_bvi_account_creation: boolean;
+    should_restrict_vanuatu_account_creation: boolean;
+    trading_platform_available_accounts: {
+        market_type: 'financial' | 'gaming';
+        name: string;
+        requirements: {
+            after_first_deposit: {
+                financial_assessment: string[];
+            };
+            compliance: {
+                mt5: string[];
+                tax_information: string[];
+            };
+            signup: string[];
+        };
+        shortcode: 'bvi' | 'labuan' | 'svg' | 'vanuatu' | 'maltainvest';
+        sub_account_type: string;
+    }[];
+    updateMT5Status: () => void;
+    fetchAccountSettings: () => void;
+    setAccountSettings: (get_settings_response: GetSettings) => void;
+    upgradeable_landing_companies: unknown[];
+    is_populating_mt5_account_list: boolean;
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    landing_companies: Record<string, any>;
+    getChangeableFields: () => string[];
+    landing_company: LandingCompany;
+    isAccountOfTypeDisabled: (account: Record<string, DetailsOfEachMT5Loginid>) => boolean;
```

Comment:
```suggestion
    isAccountOfTypeDisabled: (type: 'virtual' | 'financial' | 'gaming') => boolean;
```
@amir-deriv I believe it should be something like this 🤔 @hamza-deriv any idea? 🤔